src/main.py

```
@@ -0,0 +1,142 @@
+    s3_prefix = f'results/{transcribe_id}'
+    tasks = [process_audio_file(flow, audio_processor, s3_processor, file_path, s3_prefix) for file_path in audio_files]
+    await asyncio.gather(*tasks, return_exceptions=True)
+
+
+app = FastAPI(lifespan=lifespan)
+
+
+@app.post('/transcribe-s3')
+async def transcribe_from_s3(
+    path: str,
+    flow: 'Flow' = Depends(aqueduct_flow),
```

Ruff (0.12.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)